packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -6,19 +6,30 @@ import { iconBoldBTag } from '../icons/bold-b';
 import { iconItalicITag } from '../icons/italic-i';
 import { iconListTag } from '../icons/list';
 import { iconNumberListTag } from '../icons/number-list';
+import { errorTextTemplate } from '../patterns/error/template';
+import { iconExclamationMarkTag } from '../icons/exclamation-mark';
 
 // prettier-ignore
 export const template = html<RichTextEditor>`
-    <template>
+    <template
+        aria-disabled="${x => x.disabled}"
```

Comment:
I don't think we need to set `aria-disabled` on the host element. It is not the element with the accessibility role -- that's the tiptap editor element.

Reply:
Thanks for the explanation. Removed `aria-disabled` on the host element.